consensus/istanbul/core/backlog.go

```diff
@@ -75,10 +75,10 @@ func (c *core) checkMessage(msgCode uint64, view *istanbul.View) error {
 	return nil
 }
 
-func (c *core) storeBacklog(msg *message, src istanbul.Validator) {
-	logger := c.logger.New("from", src, "state", c.state)
+func (c *core) storeBacklog(msg *istanbul.Message, src istanbul.Validator) {
+	logger := c.logger.New("from", msg.Address, "state", c.state)
 
-	if src.Address() == c.Address() {
+	if msg.Address == c.Address() {
```

Comment:
I'd like to get rid of `src instanbul.Validator` everywhere, but didn't get around to it here b/c the backlog uses `src` as the key rather than `src.Address`